extern/config.mad.in.h

```diff
@@ -1,5 +1,4 @@
-/* config.h.  Generated by configure.  */
-/* config.h.in.  Generated from configure.ac by autoheader.  */
+/* Auto-generated config.h file powered by cmake. */
```

Comment:
First of all: I was not expecting git to utilize the xbox config file.

Second: I placed the new template file outside of the mad directory so as to minimize modifying the original files as much as possible.